helm/values.yaml

```diff
@@ -63,30 +64,56 @@ podAnnotations: {}
 deploymentSelectorMatchLabels:
   app: raw-spans-grouper
 
+statefulSetSelectorMatchLabels:
+  app: raw-spans-grouper
+
+serviceSelectorMatchLabels:
+  app: raw-spans-grouper
+
+volumeClaimTemplates:
+  name: raw-spans-grouper-rocksdb
+  storageClassName: kafka-streams-rocksdb
+  storageRequestSize: 50Gi
+
+jmx:
+  enabled: true
+  port: 7022
+  opts: "-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.port=7022 -Dcom.sun.management.jmxremote.rmi.port=7022 -Djava.rmi.server.hostname=127.0.0.1"
```

Comment:
hopefully we can get rid of JMX https://github.com/hypertrace/raw-spans-grouper/issues/25